src/app/wallet/views/gas.nim

```diff
@@ -85,38 +84,21 @@ QtObject:
     else:
       result = $(%* { "result": "-1", "success": %success, "error": { "message": %response } })
 
-  proc gasPricePredictionsChanged*(self: GasView) {.signal.}
-
-  proc getGasPricePredictions*(self: GasView) {.slot.} =
-    self.getGasPredictions("getGasPricePredictionsResult")
-
-  proc getGasPricePredictionsResult(self: GasView, gasPricePredictionsJson: string) {.slot.} =
-    let prediction = Json.decode(gasPricePredictionsJson, GasPricePrediction)
-    self.safeLowGasPrice = fmt"{prediction.safeLow:.3f}"
-    self.standardGasPrice = fmt"{prediction.standard:.3f}"
-    self.fastGasPrice = fmt"{prediction.fast:.3f}"
-    self.fastestGasPrice = fmt"{prediction.fastest:.3f}"
-    self.gasPricePredictionsChanged()
-
-  proc safeLowGasPrice*(self: GasView): string {.slot.} = result = ?.self.safeLowGasPrice
-  QtProperty[string] safeLowGasPrice:
-    read = safeLowGasPrice
-    notify = gasPricePredictionsChanged
-
-  proc standardGasPrice*(self: GasView): string {.slot.} = result = ?.self.standardGasPrice
-  QtProperty[string] standardGasPrice:
-    read = standardGasPrice
-    notify = gasPricePredictionsChanged
-
-  proc fastGasPrice*(self: GasView): string {.slot.} = result = ?.self.fastGasPrice
-  QtProperty[string] fastGasPrice:
-    read = fastGasPrice
-    notify = gasPricePredictionsChanged
-
-  proc fastestGasPrice*(self: GasView): string {.slot.} = result = ?.self.fastestGasPrice
-  QtProperty[string] fastestGasPrice:
-    read = fastestGasPrice
-    notify = gasPricePredictionsChanged
+  proc gasPriceChanged*(self: GasView) {.signal.}
+
+  proc getGasPrice*(self: GasView) {.slot.} =
+    if not self.status.wallet.isEIP1559Enabled():
```

Comment:
Is that still valid? Now that we have reach the block I'd assume this is always enabled?

Reply:
It's still valid. gasPrice is used in chains that do not have EIP1559 enabled. I disable it if EIP1559 is enabled because then the gas price will not used so it is not necessary to do a request to obtain it (infura has limits in the number of requests per day, and desktop uses by default a unique infura key, so the less requests we make, the better)